### DISCUSSION TITLE
Error handling

### DISCUSSION THREAD

**Discussion by nizyumka** [bukhalo/nestjs-telegraf#987] [Q&A] [answered]
Error handling when telegram unavailable error and application crash when module is initialized?

Example app crashed
![IMAGE 2022-11-12 23:59:32](https://user-images.githubusercontent.com/29820873/201490372-bc823cba-c00d-4bd7-9aab-86c585bb406e.jpg)

**Comment by 0x467** [comment, marked as answer]
Fixed in https://github.com/0x467/nestjs-telegraf/commit/9c0286c23c707f3366f670be3c91d61758186157
